pxf/pxf-api/src/main/java/org/apache/hawq/pxf/api/examples/DummyResolver.java

```diff
@@ -0,0 +1,29 @@
+package org.apache.hawq.pxf.api.examples;
+
+import org.apache.hawq.pxf.api.OneField;
+import org.apache.hawq.pxf.api.OneRow;
+import org.apache.hawq.pxf.api.ReadResolver;
+import org.apache.hawq.pxf.api.utilities.InputData;
+import org.apache.hawq.pxf.api.utilities.Plugin;
+import java.util.LinkedList;
+import java.util.List;
+import static org.apache.hawq.pxf.api.io.DataType.INTEGER;
+import static org.apache.hawq.pxf.api.io.DataType.VARCHAR;
+
+public class DummyResolver extends Plugin implements ReadResolver {
+    private int rowNumber;
+    public DummyResolver(InputData metaData) {
+        super(metaData);
+        rowNumber = 0;
+    }
+    @Override
+    public List<OneField> getFields(OneRow row) throws Exception {
+        /* break up the row into fields */
+        List<OneField> output = new LinkedList<OneField>();
+        String[] fields = ((String) row.getData()).split(",");
```

Comment:
row can never be null here as the ReadBridge specifically check for null. None of the other resolvers have this check as well. Will add validation for other things you mentioned